src/main/java/seedu/address/model/person/Streak.java

```diff
@@ -37,7 +37,9 @@ public static Streak from(Goal goal, List<Event> meetings) {
                 .map(Event::getDate)
                 .sorted(LocalDate::compareTo)
                 .collect(Collectors.toList());
-
+        if (meetings.isEmpty()) {
+            return new Streak(0);
+        }
```

Comment:
What does this part do? Will this display streaks for people with no meetings yet?